src/Symfony/Bundle/FrameworkBundle/CacheWarmer/ConfigBuilderCacheWarmer.php

```diff
@@ -53,7 +53,7 @@ public function warmUp(string $cacheDir): array
             try {
                 $this->dumpExtension($extension, $generator);
             } catch (\Exception $e) {
-                $this->logger?->warning('Failed to generate ConfigBuilder for extension {extensionClass}.', ['exception' => $e, 'extensionClass' => $extension::class]);
+                $this->logger?->warning('Failed to generate ConfigBuilder for extension {extensionClass}.', ['exception' => $e->getMessage(), 'extensionClass' => $extension::class]);
```

Comment:
The `exception` context field is actually standardized in PSR-3 and reserved for exception _objects_.

It's probably fine to include the exception message in the log, but please don't repurpose that context field.